Library/Homebrew/requirements/maximum_macos_requirement.rb

```diff
@@ -16,4 +16,9 @@ def message
       versions newer than #{@version.pretty_name} due to an upstream incompatibility.
     EOS
   end
+
+  def display_s
+    "MaximumMacOS = #{@version}"
```

Comment:
Do you think `<=` is as understood as `<`? I wasn't sure but happy to defer to you here.